test/subset-band-name/environment.yaml

```
@@ -0,0 +1,16 @@
+name: papermill-subset-band-name
+channels:
+  - conda-forge
+  - defaults
```

does this work if you make this nodefaults?
```suggestion
  - nodefaults
```

---

Worked and pushed.